components/avatar/avatar.tsx

```diff
@@ -109,6 +111,25 @@ const InternalAvatar: React.ForwardRefRenderFunction<unknown, AvatarProps> = (pr
     ...others
   } = props;
 
+  const screens = useBreakpoint();
+  const responsiveSizeStyle: React.CSSProperties = React.useMemo(() => {
+    if (typeof size !== 'object') {
+      return {};
+    }
+
+    const newBreakpoint: Breakpoint | undefined = responsiveArray.find(screen => screens[screen]);
```

Comment:
1. i think `newBreakpoint` => `currentPoint` is better
2. there is no way `breakpoint === undefined` if useBreakpoint api is not change. maybe you can use `responsiveArray.find(screen => screens[screen])!`